src/components/GobusterTool/Gobuster.tsx

```diff
@@ -10,7 +10,15 @@ import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/O
 import InstallationModal from "../InstallationModal/InstallationModal";
 
 const title = "GoBuster Directory and File Brute-Forcing Tool";
-const description_userguide = "GoBuster is a tool used for directory and file brute-forcing on web servers.";
+const description_userguide = `
+  GoBuster is a tool used for directory and file brute-forcing on web servers.
+  It helps identify hidden files and directories, exposing vulnerabilities
+  such as unauthorized access points or misconfigurations.
+
+  Step 1: Enter a target website URL (e.g., https://example.com).
+  Step 2: Specify the path to a wordlist file (e.g., /usr/share/wordlists/dirbuster/directory-list-2.3-medium.txt).
+  Step 3: Click Start GoBuster to initiate the scan.
+`;
```

Comment:
The changes you've made should not be in the `const description_userguide` field, please revert this back to what it is in the current `main` branch.

Changes to the steps that are displayed when actually using the tool should be edited in the `const description` field in the `const GoBusterTool = () => {` class. Please see the existing steps on line 49 of your current `Gobuster.tsx` edit.

At the moment, your edits are not reflected in the UI.